src/Components/CreateImageWizard/formComponents/ReviewStepTables.js

```diff
@@ -0,0 +1,112 @@
+import React from 'react';
+
+import { Panel, PanelMain } from '@patternfly/react-core';
+import {
+  TableComposable,
+  Tbody,
+  Td,
+  Th,
+  Thead,
+  Tr,
+} from '@patternfly/react-table';
+import PropTypes from 'prop-types';
+
+import { UNIT_GIB, UNIT_MIB } from '../../../constants';
+
+export const FSReviewTable = ({ ...props }) => {
+  return (
+    <Panel isScrollable>
+      <PanelMain maxHeight="30ch">
+        <TableComposable
+          aria-label="File system configuration table"
+          variant="compact"
+        >
+          <Thead>
+            <Tr>
+              <Th>Mount point</Th>
+              <Th>File system type</Th>
+              <Th>Minimum size</Th>
+            </Tr>
+          </Thead>
+          <Tbody data-testid="file-system-configuration-tbody-review">
+            {props.fsc.map((r, ri) => (
+              <Tr key={ri}>
+                <Td className="pf-m-width-30">{r.mountpoint}</Td>
+                <Td className="pf-m-width-30">xfs</Td>
+                <Td className="pf-m-width-30">
+                  {r.size}{' '}
+                  {r.unit === UNIT_GIB
+                    ? 'GiB'
+                    : r.unit === UNIT_MIB
+                    ? 'MiB'
+                    : 'KiB'}
+                </Td>
+              </Tr>
+            ))}
+          </Tbody>
+        </TableComposable>
+      </PanelMain>
+    </Panel>
+  );
+};
+
+export const PackagesTable = ({ ...props }) => {
```

Comment:
So in the parent component that renders this component, it is called like this: `<PackagesTable packages={getState()?.values['selected-packages']} />`

We could just call `const packages={getState()?.values['selected-packages']}` inside of this component itself.

I think that reduces the cognitive load needed to understand the component... When the packages come in as a prop, someone reading the code might ask themselves "well where did those packages come from?". Then they would need to refer to the parent component (a context switch) to find out that the come from the form state, then come back to this component (another context switch).